### PR TITLE
9479 dod part 2 - tests + docs

### PR DESCRIPTION
**Pull request by mrinsin**

### CODE REVIEW

File: web-client/pa11y/pa11y-docketclerk.js [hunk 1]
```diff
@@ -144,7 +144,6 @@ module.exports = [
       'wait for .confirm-initiate-save-modal to be visible',
     ],
     notes: 'checks a11y of confirm-initiate-save-modal dialog',
-    only: true,
```
Comment (rosiehsmith) [inline web-client/pa11y/pa11y-docketclerk.js:147]: 😨 omg - i want to add a lint rule for this. going to add a devex now

File: web-client/src/ustc-ui/DateInput/DatePickerComponent.jsx [hunk 2]
```diff
@@ -85,6 +85,9 @@ export const DatePickerComponent = ({
         if (month.length > 2) {
           [year, month, day] = splitDate(e.target.value);
         }
+        if (year.length < 4) {
+          year = '';
+        }
```
Comment (rosiehsmith) [inline web-client/src/ustc-ui/DateInput/DatePickerComponent.jsx:90]: curious, why not do this with joi validation?

**Review by rosiehsmith — APPROVED** [review]
Nice additions to pdf tests, combining the interactor calls

File: web-client/pa11y/pa11y-docketclerk.js (hunk 1 above)
Comment (rosiehsmith) [reply web-client/pa11y/pa11y-docketclerk.js:147]: https://trello.com/c/87mv6HQ6/1080-add-lint-rule-to-prevent-pa11y-tests-from-being-focused

File: web-client/src/ustc-ui/DateInput/DatePickerComponent.jsx (hunk 2 above)
Comment (mrinsin) [reply web-client/src/ustc-ui/DateInput/DatePickerComponent.jsx:90]: Good question! @jamesobrooks may have more insight

File: web-client/pa11y/pa11y-docketclerk.js (hunk 1 above)
Comment (mrinsin) [reply web-client/pa11y/pa11y-docketclerk.js:147]: Thank you!! great idea!

File: web-client/src/ustc-ui/DateInput/DatePickerComponent.jsx (hunk 2 above)
Comment (jamesobrooks) [reply web-client/src/ustc-ui/DateInput/DatePickerComponent.jsx:90]: @rosiehsmith, I could not find a way to use joi to require that the year was 4 digits without overriding the flexibility we have here.
https://stackoverflow.com/a/57793993

**Pull request merged**